readme.md

```diff
@@ -86,19 +86,11 @@ Instructions for using with WORKSPACE may be found in [release notes](https://gi
 
 When running `@multitool//tools/tool-name`, Bazel will execute the tool at the root of the runfiles tree due to https://github.com/bazelbuild/bazel/issues/3325.
 
-To run a tool in the current working directory, use the convenience target `@multitool//tools/tool-name:cwd`.
+It's possible to workaround this:
+- To run a tool in the current working directory, use the convenience target `@multitool//tools/tool-name:cwd`.
+- To run a tool in the Bazel module or workspace root, use the convenience target `@multitool/tools/tool-name:workspace_root`.
```

Comment:
Should there be a double slash here?

Reply:
Yeah good catch, that typo was here previously too.